test-packages/integration-tests/test/auth-flows/auth-flow.spec.ts

```diff
@@ -53,6 +53,20 @@ describe('OAuth flows', () => {
     expect(result.length).toBe(1);
   }, 60000);
 
+  xit('OAuth2Password: Fetches destination and destiantion service has token', async () => {
```

Comment:
This is disabled?

Reply:
All the tests in the file are local tests that need a JWT that is fetched manually, which is a convenient auth related local test instead of doing e2e test.

Reply:
Ah ok.